packages/usa-checkbox/src/usa-checkbox.twig

```diff
@@ -49,33 +49,4 @@
     <label class="usa-checkbox__label"
     for="check-historical-carver">George Washington Carver</label>
   </div>
-  <div class="usa-checkbox">
-    <input
-      class="usa-checkbox__input"
-      id="check-historical-dubois"
-      type="checkbox"
-      name="historical-figures"
-      value="web-dubois"
-      data-indeterminate
-      {% if disabled_state == "disabled" %}disabled{% endif %} 
-      {% if disabled_state == "aria-disabled" %}aria-disabled="true"{% endif %}
-    >
-    <label class="usa-checkbox__label"
-    for="check-historical-dubois">W.E.B. DuBois</label>
-  </div>
-  <div class="usa-checkbox">
-    <input
-      class="usa-checkbox__input"
-      id="check-historical-marshall"
-      type="checkbox"
-      name="historical-figures"
-      value="thurgood-marshall"
-      checked
-      data-indeterminate
-      {% if disabled_state == "disabled" %}disabled{% endif %} 
-      {% if disabled_state == "aria-disabled" %}aria-disabled="true"{% endif %}
-    >
-    <label class="usa-checkbox__label"
-    for="check-historical-marshall">Thurgood Marshall</label>
-  </div>
```

Comment:
Same case as above. This was a new addition in feature.

Reply:
Resolved in [34b7f68](https://github.com/uswds/uswds/pull/5650/commits/34b7f68fcfe709db30d05d025b6f932d3f0ca921)

Reply:
Resolved in 34b7f68.